### ISSUE TITLE
Not able to run a custom router-based handler on Ubuntu 22.04

### ISSUE THREAD

**Issue by saviorand** [saviorand/lightbug_http#31]
As reported by bitcloudo on Discord.
If running [this (awesome) project](https://github.com/bitcloud2/battlesnake-mojo) with Lightbug from Dockerfile, some cryptic errors are thrown, even though the packaging step seems to work. This only seems to happen on Ubuntu? At least I can't reproduce on my Mac locally

```
FROM ubuntu:22.04

RUN apt-get update \
    && apt-get install -y \
    wget \
    curl \
    libedit2 \
    git

# Download/Install minicoda py3.11 for linux x86/x64.
RUN mkdir -p /opt/conda 
RUN wget https://repo.anaconda.com/miniconda/Miniconda3-py311_24.1.2-0-Linux-x86_64.sh -O /opt/conda/miniconda.sh \ 
    && bash /opt/conda/miniconda.sh -b -p /opt/miniconda

ENV PATH=/opt/miniconda/bin:$PATH
RUN conda init

# Install Modular/Mojo
RUN curl -s https://get.modular.com | sh -
RUN modular auth $sometoken
RUN modular install mojo

ARG MODULAR_HOME="/root/.modular"
ENV MODULAR_HOME=$MODULAR_HOME
ENV PATH="$PATH:$MODULAR_HOME/pkg/packages.modular.com_mojo/bin"

# Compile lightbug repo
# Clone the lightbug repository
RUN git clone --depth 1 --branch latest-build https://github.com/saviorand/lightbug_http.git ./lightbug_http

# Optionally copy other necessary directories if they are not in the Git repository
COPY ./external /external
# Run the mojo package command to create the package. Ensure it is placed in a known directory.
RUN cd /lightbug_http && mojo package lightbug_http -o lightbug_http.mojopkg

# Install app
COPY . /usr/app
WORKDIR /usr/app

# Instead of using COPY, use RUN to move or copy the file within the container
RUN cp /lightbug_http/lightbug_http.mojopkg ./lightbug_http.mojopkg
# Run Battlesnake
```

**Comment by bitcloud2** [comment]
I updated to the newly released Mojo 24.3.0 (9882e19d) and retried utilizing the the packaged form of Lightbug within my repo. It now works!!! I used the lightbug directories directly in my repo (old work around), the locally packaged version of lightbug, as well as downloading the latest-build package directly from the lightbug_http repo. All three worked for me locally. I'm assuming the issue had something to do with the Mojo compiler. I still need to try this within docker, but i would assume this issue is now resolved. Thanks again for your efforts here.
